pkgs/tools/package-management/akku/overrides.nix

```diff
@@ -0,0 +1,135 @@
+{ stdenv, lib, akku, curl, git, substituteAll }:
+let
+  joinOverrides =
+    overrides: pkg: old:
+    lib.attrsets.mergeAttrsList (map (o: o pkg old) overrides);
+  addToBuildInputs =
+    extras: pkg: old:
+    { propagatedBuildInputs = old.propagatedBuildInputs ++ extras; };
+  broken = lib.addMetaAttrs { broken = true; };
+  skipTests = pkg: old: { doCheck = false; };
+  # debugging
+  showLibs = pkg: old: { preCheck = "echo $CHEZSCHEMELIBDIRS"; };
+  runTests = pkgs: old: { doCheck = true; };
+  brokenOnAarch64Darwin = pkgs: old: { broken = stdenv.isDarwin && stdenv.isAarch64; };
+  brokenOnx86_64Darwin = pkgs: old: { broken = stdenv.isDarwin && stdenv.isx86_64; };
+in
+{
+  chez-srfi = joinOverrides [
```

Comment:
This is also broken on aarch64-linux...

Reply:
I marked it as broken on Aarch64 as a whole, then